src/webhook_launcher/app/misc.py

```diff
@@ -46,21 +47,28 @@ def body_callback(self, buf):
         self.contents += buf
 
     def api_call(self, endpoint, call):
-        c = pycurl.Curl()
-        c.setopt(pycurl.SSL_VERIFYPEER, 0)
-        c.setopt(pycurl.SSL_VERIFYHOST, 0)
+        proxies = {}
+        auth = None
         if settings.OUTGOING_PROXY:
-            c.setopt(pycurl.PROXY, settings.OUTGOING_PROXY)
-            c.setopt(pycurl.PROXYPORT, settings.OUTGOING_PROXY_PORT)
-        c.setopt(pycurl.NETRC, 1)
```

Comment:
Yes, and settings reads them from the conf file, under [web] bb_api_user and bb_api_password